### PR TITLE
Update setup.py with python_requires < 3.9 -> 3.10

### PR DESCRIPTION
**Pull request by voletiv**
Does it not work with Python 3.9?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def main():
     install_requires=[
         'dill>=0.3.2',
     ],
-    python_requires='>=3.6, <3.9',
+    python_requires='>=3.6, <3.10',
```
Comment (JCGoran) [inline setup.py:48]: Seems to work fine on `3.10.6` as well, so it's probably better to just set `<3.11` (as that's the current preview version) for now.

Comment (odashi) [reply setup.py:48]: +1. it looks good to pin 3.11 for now.

**Comment by odashi** [comment]
@voletiv Thanks for proposing a PR!
I guess #39 is maybe a better fix that covers things in this PR too.

**Comment by odashi** [comment]
Close this issue since the same change has been merged.

**Pull request closed without merging**